corehq/apps/groups/templates/groups/all_groups.html

```diff
@@ -45,43 +32,94 @@
-        {% if all_groups %}
-            <h2>{% trans "Select a group:" %}</h2>
-        {% else %}
-            <h2>{% trans "Create your first group:" %}</h2>
-        {% endif %}
-    {% endblock %}
+    <p>
+        {% blocktrans %}
+            Groups are a useful way of combining multiple <strong>Mobile Workers</strong>
+            into one entity that can be managed in the following ways:
+        {% endblocktrans %}
+    </p>
+    <ul>
+        <li>
+            {% blocktrans %}
+            <strong>Reporting Groups</strong> are used in reports to filter and view data.
+            {% endblocktrans %}
+        </li>
+        <li>
+            {% blocktrans %}
+            <strong>Case Sharing</strong> allow their members to share a case list in a case-sharing app.
+            {% endblocktrans %}
+        </li>
+    </ul>
+    <form class="well well-sm form-inline" method="post" action="{% url "add_group" domain %}" id="create_group_form">
+        {% csrf_token %}
+        <input type="text" placeholder="{% trans "Group Name" %}" id="id_group_name" name="group_name" class="form-control" />
+        <button class="btn btn-success" type="submit">
+            <i class="fa fa-plus"></i>
+            {% trans "Create Group" %}
+        </button>
+    </form>
 
-    <div class="tabbable tabs-left row-fluid">
-        <ul class="nav nav-tabs span2">
-            {% with group as this_group %}
-                {% for group in all_groups %}
-                    <li {% if this_group.get_id == group.get_id %}class="active"{% endif %}>
-                        <a href="{% url "group_members" domain group.get_id %}">{{ group.display_name }}
-                            {% if group.reporting %}
-                                <i class="icon-list-alt reporting-group-icon"></i>
-                            {% else %}
-                               <i class="icon-white"></i>
-                            {% endif %}
-                            {% if group.case_sharing %}<i class="icon-star case-sharing-group-icon"></i>{% endif %}
-                        </a>
-                    </li>
-                {% endfor %}
-            {% endwith %}
-        </ul>
-        {% if not group.is_deleted %}
-        <div class="tab-content span9">
-            {% block group-view %}
-            {% endblock %}
+    <div class="panel panel-default">
+        <div class="panel-heading">
+            <h3 class="panel-title">{% trans "Project Groups" %}</h3>
+        </div>
+        <div class="panel-body">
+            {% if all_groups %}
+                <table class="table table-striped">
+                    <thead>
+                        <th>{% trans "Group Name" %}</th>
+                        <th>
+                            {% trans "Reporting Group" %}
+                            <span class="hq-help-template"
+                                data-title="{% trans 'Reporting Groups' %}"
+                                data-bootstrap3="true"
```

Comment:
What's this flag for? I don't see it used in `transformHelpTemplate` or `makeHqHelp`. I see it isn't new in this PR, just haven't encountered it before.